### PR TITLE
0.9.0 Release Proposal

### PR DESCRIPTION
**Pull request by DominicKramer**
0.9.0 Release Proposal

### CODE REVIEW

**Comment by DominicKramer** [comment]
PTAL

**Comment by ofrobots** [comment]
Nit: notable changes come before the commit list. Otherwise LGTM.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
 # Node.js Agent for Google Cloud Debug ChangeLog
 
+## 2016-11-03, Version 0.8.6 (Experimental), @dominicdkramer
```
Comment (matthewloring) [inline CHANGELOG.md]: The source map changes are probably be semver major, maybe 0.9.0 is better.

Comment (DominicKramer) [inline CHANGELOG.md]: Fixed

**Pull request merged**